src/d_deh.c

```diff
@@ -1874,6 +1876,10 @@ void deh_procThing(DEHFILE *fpin, FILE* fpout, char *line)
         {
           if (!strcasecmp(key,deh_mobjinfo[ix]))  // killough 8/98
             {
+              if (!deh_set_blood_color && !strcasecmp(key, "Blood color"))
+                {
+                  deh_set_blood_color = true;
```

Comment:
Perhaps we need more sophisticated detection of dehacked monsters?

Reply:
Maybe a new field in `mobj_t`?

Reply:
Or new flag? `MIF_DEH_COLOREDBLOOD`